.github/workflows/staging_build.yml

```
@@ -562,7 +562,7 @@ jobs:
           password: "${{ secrets.quay_registry_password }}"
 
       - name: "STAGING: Upload Knative CLI Workflow Plugin for Linux"
```

Just like other similar steps, consider adding this suffix for the name.
```suggestion
      - name: "STAGING: Upload Knative CLI Workflow Plugin for Linux (Ubuntu only)"
```